hotsos/defs/tests/scenarios/openvswitch/ovn/ovn_elections.yaml

```diff
@@ -0,0 +1,51 @@
+      2023-05-06T17:01:05.290Z|20107|raft|INFO|term 2922: 1266 ms timeout expired, starting election
+      2023-05-06T17:01:21.685Z|19956|raft|INFO|term 2923: 1338 ms timeout expired, starting election
+      2023-05-06T17:01:58.065Z|20142|raft|INFO|term 2926: 1154 ms timeout expired, starting election
+      2023-05-06T17:02:14.646Z|19988|raft|INFO|term 2927: 1204 ms timeout expired, starting election
+      2023-05-06T17:02:40.497Z|16430|raft|INFO|term 2929: 1258 ms timeout expired, starting election
+      2023-05-06T17:03:11.530Z|20024|raft|INFO|term 2931: 1175 ms timeout expired, starting election
+      2023-05-06T17:03:24.486Z|16460|raft|INFO|term 2932: 1226 ms timeout expired, starting election
+    var/log/ovn/ovsdb-server-sb.log: |
+      2023-05-06T16:56:11.414Z|19755|raft|INFO|term 2873: 1620 ms timeout expired, starting election
+      2023-05-06T16:56:13.188Z|19590|raft|INFO|term 2874: 1138 ms timeout expired, starting election
+      2023-05-06T16:56:39.219Z|16087|raft|INFO|term 2879: 1017 ms timeout expired, starting election
+      2023-05-06T16:56:52.926Z|19823|raft|INFO|term 2883: 1169 ms timeout expired, starting election
+      2023-05-06T16:57:11.497Z|19689|raft|INFO|term 2887: 1451 ms timeout expired, starting election
+      2023-05-06T16:57:35.669Z|16207|raft|INFO|term 2893: 1377 ms timeout expired, starting election
+      2023-05-06T16:58:05.309Z|19921|raft|INFO|term 2900: 1020 ms timeout expired, starting election
+      2023-05-06T16:58:09.304Z|19797|raft|INFO|term 2901: 1020 ms timeout expired, starting election
+      2023-05-06T16:58:43.221Z|16277|raft|INFO|term 2907: starting election
+      2023-05-06T16:59:11.259Z|19994|raft|INFO|term 2913: 1361 ms timeout expired, starting election
+      2023-05-06T16:59:44.511Z|19907|raft|INFO|term 2915: 1151 ms timeout expired, starting election
+      2023-05-06T16:59:54.758Z|20045|raft|INFO|term 2916: 1139 ms timeout expired, starting election
+      2023-05-06T16:59:59.892Z|16372|raft|INFO|term 2917: 1274 ms timeout expired, starting election
+      2023-05-06T17:00:25.937Z|19934|raft|INFO|term 2919: 1143 ms timeout expired, starting election
+      2023-05-06T17:00:55.262Z|16396|raft|INFO|term 2921: 1050 ms timeout expired, starting election
+      2023-05-06T17:01:05.290Z|20107|raft|INFO|term 2922: 1266 ms timeout expired, starting election
+      2023-05-06T17:01:21.685Z|19956|raft|INFO|term 2923: 1338 ms timeout expired, starting election
+      2023-05-06T17:01:58.065Z|20142|raft|INFO|term 2926: 1154 ms timeout expired, starting election
+      2023-05-06T17:02:14.646Z|19988|raft|INFO|term 2927: 1204 ms timeout expired, starting election
+      2023-05-06T17:02:40.497Z|16430|raft|INFO|term 2929: 1258 ms timeout expired, starting election
+      2023-05-06T17:03:11.530Z|20024|raft|INFO|term 2931: 1175 ms timeout expired, starting election
+      2023-05-06T17:03:24.486Z|16460|raft|INFO|term 2932: 1226 ms timeout expired, starting election
```

Comment:
as it is this will fail because under-the-hood the search constraints you have a applied are saying logs must be 48 hours old and that will use the CLIHelper.date output which will be None since your data_root does not have a sos_commands/date/date. So you have two options; either do

````
data-root:
....
  copy-from-original:
    sos_commands/date/date
```

which will give you "Thu Feb 10 16:19:17 UTC 2022" and therefore you will need to adjust the timestamps in your logs to be within 48 hours of this or you can create a fake one e.g.:

```
data-root:
  files:
    sos_commands/date/date: |
      Thu May 07 16:19:17 UTC 2023
```